### PR TITLE
update translation greedy-algorithms

### PR DESCRIPTION
**Pull request by CleoAguiar**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/portuguese/algorithms/greedy-algorithms/index.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ Você deve ter ouvido falar sobre muitas técnicas de design algorítmico enquan
...
 
 ## Definição formal
 
-Suponha que você tenha uma função objetiva que precisa ser otimizada (maximizada ou minimizada) em um determinado ponto. Um algoritmo Greedy faz escolhas gananciosas em cada etapa para garantir que a função objetivo seja otimizada. O algoritmo Greedy tem apenas uma chance para calcular a solução ótima, para que ela nunca retorne e reverta a decisão.
+Suponha que você tenha uma função objetiva que precisa ser otimizada (maximizada ou minimizada) em um determinado ponto. Um algoritmo Greedy faz escolhas gananciosas em cada etapa para garantir que a função objetivo seja otimizada. O algoritmo ganancioso tem apenas uma chance para calcular a solução ótima, para que ela nunca retorne e reverta a decisão.
```
Comment (gugacavalieri) [inline guide/portuguese/algorithms/greedy-algorithms/index.md]: Wouldn't be better to use `guloso` to match name with the rest of the file ?

File: guide/portuguese/algorithms/greedy-algorithms/index.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ Você deve ter ouvido falar sobre muitas técnicas de design algorítmico enquan
 *   Programação gananciosa
 *   Programaçao dinamica para nomear alguns. Neste artigo, você aprenderá sobre o que é um algoritmo guloso e como você pode usar essa técnica para resolver muitos problemas de programação que, de outra forma, não parecem triviais.
```
Comment (gugacavalieri) [inline guide/portuguese/algorithms/greedy-algorithms/index.md]: Here we are calling it `algoritmo guloso`

File: guide/portuguese/algorithms/greedy-algorithms/index.md [hunk 3]
```diff
@@ -71,11 +71,11 @@ function interval_scheduling_problem(requests)
  end 
 ```
 
-## Quando usamos Algoritmos Greedy?
+## Quando usamos Algoritmos Ganancioso?
```
Comment (gugacavalieri) [inline guide/portuguese/algorithms/greedy-algorithms/index.md]: same in here :)

What do you think ?

**Review by CleoAguiar — COMMENTED** [review]
Changed `guloso` to match name with the rest of the file. Thank you for review.

**Review by gugacavalieri — APPROVED** [review]
LGTM :)

**Comment by RandellDawson** [comment]
@CleoAguiar Thank you for  creating this PR.  However, since it was created, the underlying document has changed.  Can you please review and resolve the current merge conflicts?  It could be the changes to the underlying document are so significant that your changes are no longer relevant.  If the latter is the case, it might be easier to just close this PR and create a new PR with any changes you still feel are important to make.

If you need any assistance or have questions about resolving the merge conflicts, please reply back or contact someone in the [Contributors Room on Gitter](https://gitter.im/FreeCodeCamp/Contributors).

Thank you.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**